samples/subsys/mgmt/updatehub/Kconfig

```diff
@@ -20,4 +20,13 @@ config UPDATEHUB_POLLING
 	  intervention.
 endif #!UPDATEHUB_MANUAL
 
+if WIFI
+config UPDATEHUB_WIFI_SSID
```

Comment:
In the networking side, if a sample app needs a kconfig option, I have added `SAMPLE` to the option name like `CONFIG_NET_SAMPLE_....`. This way there is no confusion what options are for the sample and what to the actual functionality.
So here I would recommend to name the option `config UPDATEHUB_SAMPLE_WIFI_SSID` etc.